synapse/rest/media/v1/_base.py

```diff
@@ -197,3 +199,64 @@ def __init__(self, server_name, file_id, url_cache=False,
         self.thumbnail_height = thumbnail_height
         self.thumbnail_method = thumbnail_method
         self.thumbnail_type = thumbnail_type
+
+
+def get_filename_from_headers(headers):
+    """
+    Get the filename of the downloaded file by inspecting the
+    Content-Disposition HTTP header.
+
+    Args:
+        headers (twisted.web.http_headers.Headers): The HTTP
+        request headers.
```

Comment:
```suggestion
            request headers.
```

Reply:
It would be nice if we had a docstring format that tools could autoformat for us...